lib/ExpensiMark.js

```diff
@@ -453,7 +457,31 @@ export default class ExpensiMark {
 
                     return `!(${g2})`;
                 }
-            }
+            },
+            {
+                name: 'roomMention',
```

Comment:
Now I wonder if we should keep the `room` here or change it to the `report`

Reply:
change to report :P